config/locales/hyrax.en.yml

```diff
@@ -257,13 +257,21 @@ en:
       workflow_roles:
         header: Workflow Roles
         index:
+          current_group_roles: Current Group Roles
           current_roles: Current Roles
+          current_user_roles: Current User Roles
```

Comment:
remove line 262 for this and all other languages

Reply:
Removed lines 262 for all languages.